src/e2ir.d

```diff
@@ -4411,7 +4411,7 @@ elem *toElem(Expression e, IRState *irs)
                 case X(Tfloat64,Tcomplex32):
                 case X(Tfloat64,Tcomplex64):
                 case X(Tfloat64,Tcomplex80):
-                    e = el_bin(OPadd,TYcfloat,el_long(TYidouble,0),e);
+                    e = el_bin(OPadd,TYcdouble,el_long(TYidouble,0),e);
```

Comment:
Why is cdouble more correct that cfloat?  This seems to cover float, double and real.